contrib/ffmpeg/module.defs

```diff
@@ -57,9 +57,11 @@ FFMPEG.CONFIGURE.extra = \
     --enable-encoder=ffv1 \
     --enable-demuxer=av1 \
     --enable-demuxer=obu \
+    --enable-demuxer=vvc \
     --enable-decoder=av1 \
     --enable-libdav1d \
     --enable-decoder=libdav1d \
+    --enable-decoder=vvc \
```

Comment:
We already enable all decoders, so there is no need to enable it manually. VVC decoding is already working since 1.8, probably those various --enable-decoder and --enable-demuxer about AV1 can be re removed too.

Reply:
ohh, that is why I was confused since explicit `--enable-decoder` and `--enable-demuxer` are used multiple times in configuration